zookeeper/package/install.sls

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would normally name this directory `zookeeper/archive/install.sls` because these SLS files are handling archive file  not linux-native packaging.  However its not a community convention, although I have promoted that convention: https://github.com/saltstack-formulas/template-formula/pull/215

Reply:
Ok. Thanks. I can change that for this and the nifi formula when I have more time.